logstash-core/src/main/java/org/logstash/health/HealthObserver.java

```diff
@@ -19,10 +19,22 @@
 package org.logstash.health;
 
 import com.google.common.collect.Iterables;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 import java.util.EnumSet;
 
 public class HealthObserver {
+
+    private static final Logger LOGGER = LogManager.getLogger();
+
+    private final MultiIndicator rootIndicator = new MultiIndicator();
+    private final MultiIndicator pipelinesIndicator = new MultiIndicator();
+
+    public HealthObserver() {
+        this.rootIndicator.attachIndicator("pipelines", this.pipelinesIndicator);
+    }
+
     public final Status getStatus() {
         // INTERNAL-ONLY Proof-of-concept to show flow-through to API results
         switch (System.getProperty("logstash.apiStatus", "green")) {
```

Comment:
Don't we need to reflect `health_report` high level status by default? NOW: from the user experience, one caveat would be with `/_health_report` API we get yellow status but `/_node/stats` shows green, they mismatch.